WooCommerce/Classes/AppDelegate.swift

```diff
@@ -263,7 +263,7 @@ private extension AppDelegate {
     ///
     func setupNoticePresenter() {
         var noticePresenter = ServiceLocator.noticePresenter
-        noticePresenter.presentingViewController = window?.rootViewController
+        noticePresenter.presentingViewController = appCoordinator?.tabBarController
```

Comment:
Comes from #3517 – fix included in release branch as we wanted to include it in 5.9 but only got ready after I did the cut of the release branch, but before had time to finished the release cut process (i.e. while I was waiting for the CocoaPods CDN to sync)